src/tests/components/App.test.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import App from "../../components/App";
+
+describe("App", () => {
+  it("renders correctly", () => {
+    const { asFragment } = render(<App />);
+    expect(asFragment()).toMatchSnapshot();
```

Comment:
Its convention to leave a blank line above the block of assertions in your tests, just helps with general readability